source/end-user-guide/preferences/connect-multiple-workspaces.rst

```diff
@@ -132,3 +129,20 @@ Granting a system permission sets it to **Accept**, and revoking it sets it to *
   - You can't manage system permissions when using the Mattermost mobile app.
   - You'll be prompted to accept or deny notifications after adding a new server connection, and any time you open the desktop app if you haven't explicitly accepted or denied system permissions.
   - You may also need to enable notifications for Mattermost within your operating system preferences.
+
+Open multiple workspace contexts
+---------------------------------
+
+From desktop v6.0, you can keep multiple workspaces open at the same time and work across them without constant switching to improve situational awareness and operational efficiency. This capability is especially useful for organizations that operate in distributed or multi-domain environments, as it enhances cross-system interoperability and command-level situational awareness.
+
+Select the **New tab** |plus| icon at the top of the desktop app to open a new tab for the current workspace. Workspace tabs have their own notifications and badges. You can drag and drop to reorder the tabs in the main desktop window.
```

Comment:
> Workspace tabs have their own notifications and badges

The notification badge is on the server switcher right now, in the future I'm hoping to give each tab its own notification badge, but for now it's just on the server switcher.